wiiu/vino/vino.ts

```diff
@@ -354,31 +441,31 @@ export default interface VinoAPI {
 	lyt_startTouchEffectToFocused: () => void;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
+	 * Shows a touch effect on the screen manually
+	 * @param left - Left position in screen
+	 * @param top -  Top position in screen
 	 */
-	lyt_startTouchEffectByPos: (unk1: number, unk2: number) => void;
+	lyt_startTouchEffectByPos: (left: number, top: number) => void;
 
 	/**
 	 * Unknown
 	 */
 	lyt_startTouchNodeEffectToFocused: () => void;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
-	 * @param unk3 - Unknown
-	 * @param unk4 - Unknown
+  	 * Shows a blue glow effect similar to the Internet Browser blue hover
+	 * @param left - Left position in screen
+	 * @param top -  Top position in screen
+	 * @param width - Width of the layout box on screen
+  	 * @param height - Height of the layout box on screen
```

Comment:
Same comments as the previous method which lised `in screen`, there's extra spaces and "in screen" doesn't make much sense